src/Tizen.Applications.Cion/Tizen.Applications/Cion/SecurityInfo.cs

```diff
@@ -0,0 +1,168 @@
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set CA path.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the cert path.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when the cert path is invalid.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public string CertPath
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecurityGetCertPath(_handle, out string certPath);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get cert path.");
+                    return "";
+                }
+                return certPath;
+            }
+
+            set
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecuritySetCertPath(_handle, value);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set cert path.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the private key path.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when the private key path is invalid.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public string PrivateKeyPath
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecurityGetPrivateKeyPath(_handle, out string privateKeyPath);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get private key path.");
+                    return "";
+                }
+                return privateKeyPath;
+            }
+
+            set
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecuritySetPrivateKeyPath(_handle, value);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set private key path.");
+                }
+            }
+        }
+
+        #region IDisposable Support
+        private bool disposedValue = false;
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
+        /// </summary>
+        /// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>
+        /// <since_tizen> 9 </since_tizen>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
+            {
+                if (disposing)
+                {
+                }
+                _handle.Dispose();
```

Comment:
_handle이 managed  object이기 때문에 disposing = true 일때만 Dispose해야 합니다.
disposing이 false인 경우는 GC에의해서 호출된 경우이며, 이 경우는 _handle이 이미 GC에 의해서 release된 후일 수 있습니다.
```suggestion
                if (disposing)
                {
                    _handle.Dispose();
                }
```

Reply:
반대로 해놨었네요. 수정했습니다.